lib/src/embedded/isolate_dispatcher.dart

```diff
@@ -156,19 +156,28 @@ class IsolateDispatcher {
     var receivePort = ReceivePort();
     isolate.sink.add((receivePort.sendPort, compilationId));
 
-    var channel = IsolateChannel<Uint8List?>.connectReceive(receivePort)
-        .transform(const ExplicitCloseTransformer());
-    channel.stream.listen(_channel.sink.add,
+    var channel = IsolateChannel<Uint8List>.connectReceive(receivePort);
+    channel.stream.listen(
+        (message) {
+          _channel.sink.add(Uint8List.sublistView(message, 1));
```

Comment:
If we're keeping the first byte as a special thing between the isolate and the main thread, then I think it would be helpful to add a comment immediately before this line about why we're sending a slice of the bytearray.